src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -364,6 +365,38 @@ public void onBlockDropItems(BlockDropItemEvent e) {
                     }
                 }
             }
+
+            // Handle double drops for Farmer Talisman
+            if (SlimefunTag.FARMER_TALISMAN_TRIGGERS.isTagged(type)) {
+                Collection<Item> drops = e.getItems();
+
+                if (Talisman.trigger(e, SlimefunItems.TALISMAN_FARMER, false)) {
+                    int dropAmount = getAmountWithFortune(type, meta.getEnchantLevel(Enchantment.LOOT_BONUS_BLOCKS));
+
+                    // Keep track of whether we actually doubled the drops or not
+                    boolean doubledDrops = false;
+
+                    // Loop through all dropped items
+                    for (Item drop : drops) {
+                        ItemStack droppedItem = drop.getItemStack();
+
+                        // We do not want to dupe blocks
+                        if (!droppedItem.getType().isBlock()) {
+                            int amount = Math.max(1, (dropAmount * 2) - droppedItem.getAmount());
+                            e.getBlock().getWorld().dropItemNaturally(e.getBlock().getLocation(), new CustomItemStack(droppedItem, amount));
+                            doubledDrops = true;
+                        }
+                    }
+
+                    if (doubledDrops) {
+                        Talisman talisman = SlimefunItems.TALISMAN_FARMER.getItem(Talisman.class);
+
+                        if (talisman != null) {
+                            talisman.sendMessage(e.getPlayer());
+                        }
+                    }
+                }
+            }
```

Comment:
This is all almost identical with the Miner Talisman code above.
To combat this redundancy, you could create a `doubleDrops` method with parameters here